requirements.txt

```diff
@@ -1,12 +1,13 @@
 astral==3.2
+Babel==2.11.0
 cachetools==5.2.0
 cairocffi==1.4.0
 CairoSVG==2.5.2
 caldav==0.10.0
 certifi==2022.9.24
 cffi==1.15.1
 charset-normalizer==2.1.1
-cryptography==38.0.2
+cryptography==38.0.1
```

Comment:
38.0.2 was yanked, so going one version behind.  I tried with 39.0.1 but that wouldn't install at all, it seemed to require the Rust compiler for some reason??